build.zig

```diff
@@ -43,6 +43,10 @@ pub fn build(b: *std.Build) !void {
     const deps = build_helpers.generateModuleDependencies(
         b,
         &external_dependencies,
+        .{
+            .optimize = optimize,
+            .target = target,
```

Comment:
no its using for all deps that we include, same for test, but if u run test in debug mode(as u do it always) all deps will debug mode too.
i need it due perfomance of deps, for generating keys

Reply:
can you link the part of the doc where they explain that when you run `zig build test` it will ignore the `.optimize` flag you explicitly set and use the debug one instead?